patroni/api.py

```diff
@@ -1014,8 +1015,9 @@ def do_POST_reinitialize(self) -> None:
             logger.debug('received reinitialize request: %s', request)
 
         force = isinstance(request, dict) and parse_bool(request.get('force')) or False
+        from_leader = isinstance(request, dict) and parse_bool(request.get('from-leader')) or False
```

Comment:
It is better to use snake case rather then kebab case.
```suggestion
        from_leader = isinstance(request, dict) and parse_bool(request.get('from_leader')) or False
```